build/firstrun.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/sh
 
+echo
```

Comment:
we found this useful to keep firstrun from happening more than once. mostly for speed